products/BellHybrid/apps/common/include/common/widgets/LabelOption.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+
+#include <common/widgets/LabelListItem.hpp>
+#include <common/widgets/ListViewWithLabels.hpp>
+#include <common/options/OptionBellMenu.hpp>
+namespace gui::option
```

Comment:
I'd add a newline between includes and beginning of the namespace as this seems to be a standard in our codebase

Reply:
Done